common/common.go

```diff
@@ -127,6 +127,69 @@ const (
 	WINDOWS            = "windows"
 )
 
+const (
```

Comment:
make it configurable

Reply:
create a new dir with name `imagepolicy` under `pkg` , and the `Kyverno` is one of the implementations just like `buildah` did with `imageengine`.